index.qmd

```diff
@@ -3,13 +3,13 @@ title: "Cloud-Optimized Geospatial Formats Guide"
 subtitle: "Methods for Generating and Testing Cloud-Optimized Geospatial Formats"
 ---
 
-If you wish to provide optimized access to geospatial data, this guide is for you. Given the large and growing size of geospatial data, users can no longer rely solely on file download to achieve their science goals.
+If you wish to provide optimized access to geospatial data, this guide is for you. Given geospatial data’s growing size, users can no longer rely solely on file downloads to achieve their science goals.
```

Comment:
I think we should re-order these sentences, possibly have an expanded section on Why cloud optimize.

Reply:
@wildintellect, I expanded the section, though we might want to review or adapt it.